udf-examples/Dockerfile

```diff
@@ -16,62 +16,55 @@
 
 # A container that can be used to build UDF native code against libcudf
 ARG CUDA_VERSION=11.0
-ARG CUDA_SHORT_VERSION=${CUDA_VERSION}
 ARG LINUX_VERSION=ubuntu18.04
-ARG CUDF_REPO=https://github.com/rapidsai/cudf
-ARG CUDF_TAG=branch-0.20
-ARG CC=7
-ARG CXX=7
 
 FROM nvidia/cuda:${CUDA_VERSION}-devel-${LINUX_VERSION}
-ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/lib
-# Needed for cudf.concat(), avoids "OSError: library nvvm not found"
-ENV NUMBAPRO_NVVM=/usr/local/cuda/nvvm/lib64/libnvvm.so
-ENV NUMBAPRO_LIBDEVICE=/usr/local/cuda/nvvm/libdevice/
-ENV DEBIAN_FRONTEND=noninteractive
 
-ARG CC
-ARG CXX
-RUN apt update -y --fix-missing && \
-    apt upgrade -y && \
-    apt install -y --no-install-recommends \
-      git \
-      gcc-${CC} \
-      g++-${CXX} \
-      libboost-all-dev \
-      openjdk-8-jdk \
-      maven \
-      tzdata && \
-    apt-get autoremove -y && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+ARG DEBIAN_FRONTEND=noninteractive
+ARG PARALLEL_LEVEL=10
+ENV PARALLEL_LEVEL=10
 
-# Install conda
-ADD https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh /miniconda.sh
-RUN sh /miniconda.sh -b -p /conda && /conda/bin/conda update -n base conda
-ENV PATH=${PATH}:/conda/bin
-# Enables "source activate conda"
-SHELL ["/bin/bash", "-c"]
+RUN GCC_VERSION=$(bash -c '\
+CUDA_VERSION=$(nvcc --version | head -n4 | tail -n1 | cut -d" " -f5 | cut -d"," -f1); \
```

Comment:
nit: having CUDA_VERSION in the shell script can be a little confusing with it is also an ARG to the Dockerfile.

Reply:
If it's just a minor nit, I'd like to keep this in-step with the original RAPIDS example since this came verbatim from there.